### PR TITLE
first go at revised clock-trigger tutorial

### LINKED ISSUE #61: tutorial: upgrade clock-triggers => xtriggers
https://github.com/metomi/rose/blame/2018.06.0/sphinx/tutorial/cylc/furthertopics/clock-triggered-tasks.rst#L9

### PR DESCRIPTION
**Pull request by wxtim**
Should Close #61 
Questions:
- I'm I emphasizing that clock triggers are a special case of xtriggers (gateway). Good Idea?
- I don't go into specifics or much detail about `my_trigger = wall_clock(offset=PT0H):<ISOdatetime freq>`. Should I?

### CODE REVIEW

**Comment by oliver-sanders** [comment]
Looks good.

**Comment by wxtim** [comment]
Failing in the same way as master. 😞

File: src/tutorial/furthertopics/clock-triggered-tasks.rst [hunk 1]
```diff
@@ -44,11 +44,11 @@ Within your ``~/cylc-run`` directory create a new directory called
...
    cd ~/cylc-run/clock-trigger
 
-Paste the following code into a :cylc:conf:`flow.cylc` file:
+Paste the following code into a ``flow.cylc`` file:
 
 .. code-block:: cylc
 
-   [scheduler]
+   [cylc]
        UTC mode = True # Ignore DST
```
Comment (MetRonnie) [inline src/tutorial/furthertopics/clock-triggered-tasks.rst:52]: Mistake?

File: src/tutorial/furthertopics/clock-triggered-tasks.rst [hunk 2]
```diff
@@ -44,11 +44,11 @@ Within your ``~/cylc-run`` directory create a new directory called
    mkdir ~/cylc-run/clock-trigger
    cd ~/cylc-run/clock-trigger
 
-Paste the following code into a :cylc:conf:`flow.cylc` file:
+Paste the following code into a ``flow.cylc`` file:
 
 .. code-block:: cylc
 
-   [scheduler]
+   [cylc]
```
Comment (wxtim) [inline src/tutorial/furthertopics/clock-triggered-tasks.rst:51]:
```suggestion
   [scheduler]
```

Comment (wxtim) [inline src/tutorial/furthertopics/clock-triggered-tasks.rst]:
```suggestion
    [scheduler]
```

**Review by MetRonnie — APPROVED** [review]
Ignoring the test failure due to the tutorial suite datapoint for now

**Review by oliver-sanders — APPROVED** [review]

**Pull request merged**